helm-charts/helm2/strimzi-kafka-operator/README.md

```diff
@@ -84,49 +84,63 @@ the documentation for more details.
 | Parameter                            | Description                               | Default                                              |
 | ------------------------------------ | ----------------------------------------- | ---------------------------------------------------- |
 | `watchNamespaces`                    | Comma separated list of additional namespaces for the strimzi-operator to watch | []             |
+| `image.registry`                     | Cluster Operator image registry           | `quay.io`                                            |
 | `image.repository`                   | Cluster Operator image repository         | `strimzi`                                            |
 | `image.name`                         | Cluster Operator image name               | `cluster-operator`                                   |
 | `image.tag`                          | Cluster Operator image tag                | `latest`                                             |
-| `image.imagePullPolicy`              | Image pull policy for all pods deployed by Cluster Operator       | `IfNotPresent`                                       |
-| `image.imagePullSecrets`              | Docker registry pull secret              | `nil`                                                 |
+| `image.imagePullPolicy`              | Image pull policy for all pods deployed by Cluster Operator       | `IfNotPresent`               |
+| `image.imagePullSecrets`             | Docker registry pull secret               | `nil`                                                |
```

Comment:
just "Image registry"? no "Docker" anymore? wdyt?

Reply:
I really just fixed the table alignment there. There are many places where we use Docker registry. So if you want to change that, I suggest to do that in separate PR.